app/controllers/trucks_controller.rb

```diff
@@ -1,5 +1,6 @@
 class TrucksController < ApplicationController
   helper_method :trucks
+  helper_method :truck
```

Comment:
Can you see if this allows you to use
`helper_method :trucks, :truck`
I feel like it might